src/RNCamera.js

```diff
@@ -399,6 +404,14 @@ export default class Camera extends React.Component<PropsType, StateType> {
       options.mute = true;
     }
 
+    if (
+      (options.mute || captureAudio) &&
+      recordAudioPermissionStatus !== RecordAudioPermissionStatusEnum.AUTHORIZED
+    ) {
+      // eslint-disable-next-line no-console
+      console.warn('Recording with audio not possible. Permissions are missing.');
```

Comment:
@sibelius Do you know a smart way to strip those warnings in the release bundle? Is there a way to mark them?

Reply:
if (__DEV__)

But you should strip all console log, there is a Babel plugin to do this

Reply:
check this https://www.npmjs.com/package/babel-plugin-transform-remove-console

Reply:
Shouldn't the metro bundler have a strip for `DEV` in release builds by default?

Reply:
not sure if __DEV__ behavior is well documented

Reply:
There is a paragraph here: https://facebook.github.io/react-native/docs/performance#profiling

I wrapped all if statements that would result in `console.warn` with `if (__DEV__) {}`